city_scrapers/spiders/losca_Health_Commission.py

```diff
@@ -0,0 +1,104 @@
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class LoscaHealthCommissionSpider(CityScrapersSpider):
+    name = "losca_Health_Commission"
+    agency = "Los Angeles Health Commission"
+    timezone = "America/Los_Angeles"
+    committee_id: int = 6
+    start_year: int = 2024
```

Comment:
- Consider making `start_year` a parameter or dynamically determining the current year to ensure the scraper remains up-to-date without manual changes.
- We usually fetch meetings in a timeframe: from 6 months ago next 12 months. This way we can scrape additional "links" (documents, meeting minutes, etc...) of the "passed" meetings.